src/Core/Commander/Interfaces/ApiInterface/ApiGlobe.js

```diff
@@ -230,6 +271,7 @@ ApiGlobe.prototype.createSceneGlobe = function(coordCarto, viewerDiv) {
     var coordinate = new GeoCoordinate().copy(coordCarto,UNIT.DEGREE);
 
     this.scene = Scene(coordinate, ellipsoid, viewerDiv, debugMode, gLDebug);
+    this.scene.parent = this;
```

Comment:
To access setNewParams inside ApiGlobe from GlobeControls. But you said that it should be in another place so it will probably change.